### PR TITLE
Feature: Setup command.

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Creates a setup command which makes the setup as simple as possible.

### CODE REVIEW

**Review by yannicklamprecht — APPROVED** [review]
LGTM

File: src/main/java/de/chojo/repbot/commands/Invite.java [hunk 1]
```diff
@@ -44,4 +45,4 @@ private MessageEmbed getResponse(MessageEventWrapper eventWrapper) {
                         Replacement.create("URL", configuration.links().invite())))
                 .build();
     }
-}
+}
```
Comment (dlsf) [inline src/main/java/de/chojo/repbot/commands/Invite.java:48]: Missing newline

File: src/main/java/de/chojo/repbot/commands/Thankwords.java [hunk 2]
```diff
@@ -376,19 +373,4 @@ private boolean loadDefaults(SlashCommandEvent slashCommandEvent) {
...
-        public List<String> get(String key) {
-            return defaults.get(key);
-        }
-
-        public Set<String> getAvailableLanguages() {
-            return Collections.unmodifiableSet(defaults.keySet());
-        }
-    }
-}
+}
```
Comment (dlsf) [inline src/main/java/de/chojo/repbot/commands/Thankwords.java:376]: Missing newline

File: src/main/java/de/chojo/repbot/serialization/ThankwordsContainer.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class ThankwordsContainer {
+    private final Map<String, List<String>> defaults = new HashMap<>();
+
+    public ThankwordsContainer() {
```
Comment (dlsf) [inline src/main/java/de/chojo/repbot/serialization/ThankwordsContainer.java:12]: Comment why this constructor is empty or remove it

File: src/main/resources/database/1/patch_3.sql [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+DELETE
+from repbot_schema.active_channel;
+
+create unique index if not exists active_channel_guild_id_channel_id_uindex
+    on repbot_schema.active_channel (guild_id, channel_id);
+
+insert into repbot_schema.active_channel(guild_id, channel_id)
+select guild_id, channel_id
+from temp_channel;
```
Comment (dlsf) [inline src/main/resources/database/1/patch_3.sql:14]: Missing newline

File: src/main/resources/locale_de_DE.properties [hunk 5]
```diff
@@ -123,23 +134,29 @@ command.thankwords.sub.loadDefault.added=Folgende Dankeswörter würden hinzugef
 command.thankwords.sub.loadDefault.available=Verfügbare Sprachen:
 command.thankwords.sub.remove=Löscht ein Muster
 command.thankwords.sub.remove.removed=Muster %PATTERN% wurde entfernt.
+conversation.cancel=Abbrechen
+conversation.canceled=Konversation abgebrochen.
+conversation.exit=beenden
```
Comment (dlsf) [inline src/main/resources/locale_de_DE.properties:139]: This should probably be uppercase?

**Review by dlsf — CHANGES_REQUESTED** [review]

File: src/main/resources/locale_de_DE.properties (hunk 5 above)
Comment (yannicklamprecht) [reply src/main/resources/locale_de_DE.properties:139]:
```suggestion
conversation.exit=Beenden
```

File: src/main/java/de/chojo/repbot/commands/Invite.java (hunk 1 above)
Comment (yannicklamprecht) [reply src/main/java/de/chojo/repbot/commands/Invite.java:48]:
```suggestion
}

```

File: src/main/java/de/chojo/repbot/commands/Thankwords.java (hunk 2 above)
Comment (yannicklamprecht) [reply src/main/java/de/chojo/repbot/commands/Thankwords.java:376]:
```suggestion
}

```

File: src/main/resources/database/1/patch_3.sql (hunk 4 above)
Comment (yannicklamprecht) [reply src/main/resources/database/1/patch_3.sql:14]:
```suggestion
from temp_channel;

```

**Review by dlsf — APPROVED** [review]

**Pull request merged**